src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperPartialExtensions.cs

```diff
@@ -241,6 +241,94 @@ public static class HtmlHelperPartialExtensions
         /// <param name="partialViewName">
         /// The name of the partial view used to create the HTML markup. Must not be <c>null</c>.
         /// </param>
+        /// <remarks>
+        /// In this context, "renders" means the method writes its output using <see cref="ViewContext.Writer"/>.
```

Comment:
It's the same comment than in the Async version. I just removed the <returns> section of the comment. Everything else seemed to apply